backend/src/main/java/ai/verta/modeldb/cron_jobs/DeleteEntitiesCron.java

```diff
@@ -488,10 +415,10 @@ private void deleteRepositories(Session session) {
           var deleteTagsHql =
               new StringBuilder("DELETE " + TagsEntity.class.getSimpleName() + " te where te.id.")
                   .append(ModelDBConstants.REPOSITORY_ID)
```

Comment:
You should make this all a string (like I mentioned above); then you won't need the `REPO_ID_QUERY_POST_APPENDER` variable at all.

Reply:
Updated to string instead of string builder and removed `REPO_ID_QUERY_POST_APPENDER`